spec/features/participant-declarations/voided_declaration_in_payment_breakdown_spec.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+require_relative "./participant_declaration_steps"
+
+RSpec.feature "Voided declaration in payment breakdown", type: :feature do
+  include ParticipantDeclarationSteps
+
+  before(:each) do
+    setup
+  end
+
+  scenario "Payment breakdown does not include voided declarations" do
+    given_an_early_career_teacher_has_been_entered_onto_the_dfe_service
+    and_call_off_contract_was_created_for_lead_provider
+    when_the_participant_details_are_passed_to_the_lead_provider
+    @declaration_id = and_the_lead_provider_submits_a_declaration_for_the_ect_using_their_id.dig("data", "id")
```

Comment:
Can we make `@ declaration_id` part of `and_the_lead_provider_submits_a_declaration_for_the_ect_using_their_id` step?